src/Files.App/Actions/Content/Install/InstallFontAction.cs

```diff
@@ -31,12 +31,10 @@ public InstallFontAction()
 			context.PropertyChanged += Context_PropertyChanged;
 		}
 
-		public Task ExecuteAsync()
+		public async Task ExecuteAsync()
 		{
 			foreach (ListedItem selectedItem in context.SelectedItems)
```

Comment:
How about a `List<Task>` is used and then `await Task.WhenAll()` is called on that list to execute all tasks concurrently?

Reply:
thank you